apps/mainsite/drf_spectacluar.py

```diff
@@ -38,17 +40,14 @@ def _contains_excluded_keywords(path):
 
 
 def custom_postprocessing_hook(result, generator, request, public):
-    result["security"] = [{"openId": []}],
+    result['security'] = ([{'openId': []}],)
```

Comment:
I presume your IDE or editor automatically formats the file automatically on save?

While I really love these consistent and small formatting improvements, they make the PR and git history a tad hard to reason about. As can also be witnessed in the CI CodeQL raising false negative security issues on such "changes". 

My *personal* preference is to put all "lintfixes" and "autoformat" fixes in a single commit, eg. at the beginning of a PR. That way we still have the very much needed and wanted consisntency fixes, but keep them out of the commits  that actually change behaviour/businesslogic etc.

Reply:
Good point, will remember it for future changes. Also, I thin we all should use the same linter and formatter, in this case Ruff ( https://docs.astral.sh/ruff/ ). A config file is already added to the repo, ruff.toml.